tests/fsharp/core/signedtests/build.bat

```diff
@@ -0,0 +1,265 @@
+@if "%_echo%"=="" echo off
```

Comment:
@KevinRansom Ok, but we need to jump here sooner or later. Maintaining two sets of test runner scripts is a recipe for trouble :) 

Which .bat files are needed for coreclr testing? All of them or some specific new ones? Thx